modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -615,6 +615,107 @@ definition {
 		}
 	}
 
+	@description = "LPS-135404 - Verify that it's possible to delete created custom fields in a System Object"
+	@ignore = "true"
+	@priority = "4"
+	test CanDeleteCustomFieldInSystemObject {
+		task ("Create Custom Fields (via API)") {
+			for (var systemNames : list "AccountEntry,CPDefinition,CommercePricingClass,CommerceOrder,Address,User") {
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Text",
+					fieldLabelName = "Custom Text",
+					fieldName = "customText",
+					fieldType = "String",
+					isRequired = "false",
+					objectName = "${systemNames}");
+			}
+		}
+
+		task ("Go to Objects, select System Object Account and click on fields tab") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Account");
+
+			ObjectAdmin.goToFieldsTab();
+		}
+
+		task ("Delete custom text field") {
+			ObjectAdmin.deleteObjectField(fieldName = "Custom Text");
+
+			ObjectAdmin.assertObjectFieldNotPresent(fieldLabel = "Custom Text");
+		}
```

Comment:
I just realized this too could be in a loop and the entire test could be two loops. However, I think this way is easier to read and follow along.